### PR TITLE
Add footer cms api

### PR DESCRIPTION
**Pull request by goelsonali**
## Footer section cms api 

Footer api with all the details 

## Screenshot 

API response -

Screenshot 2024-06-14 at 01 07 31

### CODE REVIEW

File: src/test/java/com/wcc/platform/controller/FooterControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+public class FooterControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private CmsService mockCmsService;
+
+    @Test
+    void testInternalServerError() throws Exception {
```
Comment (dricazenck) [inline src/test/java/com/wcc/platform/controller/FooterControllerTest.java:28]: Would be nice to have a success case, what do you think? I also have to create for team page.

**Review by dricazenck — APPROVED** [review]
🚀

File: src/test/java/com/wcc/platform/controller/FooterControllerTest.java (hunk 1 above)
Comment (goelsonali) [reply src/test/java/com/wcc/platform/controller/FooterControllerTest.java:28]: yeah i think i had it in my mind but just missed it will add it

Comment (goelsonali) [reply src/test/java/com/wcc/platform/controller/FooterControllerTest.java:33]: added

**Pull request merged**